### ISSUE TITLE
fine-tuning my data show no depth info on road

### ISSUE THREAD

**Issue by tresym** [tinghuiz/SfMLearner#80]
![190532](https://user-images.githubusercontent.com/19006684/38988639-3e19c2de-43d4-11e8-809f-4ba7bcc8fbc2.png)
![229817](https://user-images.githubusercontent.com/19006684/38988640-3fb28b08-43d4-11e8-988d-6801dabb19ea.png)

the first image is test on your pretrained model.
the second is fine tuned trained on my 6000 images for about 30 epochs, the road shows this behavior just few epochs of trainings.
any idea why road have no depth information? Thank you!

**Comment by tinghuiz** [comment]
This issue might be due to the lack of scale normalization during training, and has been pointed out in a recent paper: https://arxiv.org/abs/1712.00175.
